webservices/resources/candidates.py

```diff
@@ -32,6 +32,7 @@ class CandidateList(ApiResource):
     schema = schemas.CandidateSchema
     page_schema = schemas.CandidatePageSchema
     filter_multi_fields = filter_multi_fields(models.Candidate)
+    filter_fulltext_fields = [('q', models.CandidateSearch.fulltxt)]
```

Comment:
What specifically is going on here?  It looks like this is just setting a property on the class that is a list value comprising of tuples that represent each field, but I just want to be sure.  What does the `'q'` in the tuple represent in this case?